crates/revm/src/db/states/reverts.rs

```diff
@@ -93,6 +93,7 @@ impl AccountRevert {
     }
 }
 
+// mattsse: what does this exactly represent? the action that needs to be taken on revert? maybe rename?
```

Comment:
Added comment here. And will address it more in the docs: https://github.com/bluealloy/revm/issues/683